net.sourceforge.electrip.Electrip.json

```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "net.sourceforge.electrip.Electrip",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
```

Comment:
We recommend that apps use the latest version of their runtime if possible. In this case that would be 3.30. Is there a reason why you can't use that?

Reply:
[Above](https://github.com/flathub/flathub/pull/703/#pullrequestreview-171343589) @TingPing asked me the same question.
Currently the Pygame from ```shared-modules``` doesn't build in the 3.30 runtime. This is the only reason why I do not use it in the manifest. I now opened an issue [here](https://github.com/flathub/shared-modules/issues/36).

Reply:
That's good to know and is a perfectly good reason.